contrib/dev/Vagrantfile

```diff
@@ -2,23 +2,22 @@
 # vi: set ft=ruby :
 
 require 'vagrant-openstack-provider'
+require 'vagrant-reload'
+require 'vagrant-vbguest'
 
-PREPARE_BOX=ENV.fetch("PREPARE_BOX", "") == "true" ? true : false
-PROVISION_BOX=ENV.fetch("PROVISION_BOX", ENV.fetch("PREPARE_BOX", "")) == "true" ? true : false
-SYNCED_FOLDER=ENV.has_key?("GOPATH") ? ENV.fetch("GOPATH") + "/src/github.com/skydive-project/skydive" : "../.."
-
-$skydive_extra_config = {
-  "http.ws.pong_timeout" => 10,
-  "agent.topology.netlink.metrics_update" => 5,
-  "analyzer.listen" => "0.0.0.0:8082",
-  "analyzer.flow.backend" => "elasticsearch",
-  "analyzer.topology.backend" => "elasticsearch",
-  "storage.elasticsearch.host" => "127.0.0.1:9200"
-}
+PREPARE_BOX = ENV.fetch("PREPARE_BOX", "") == "true" ? true : false
+PROVISION_BOX = ENV.fetch("PROVISION_BOX", ENV.fetch("PREPARE_BOX", "")) == "true" ? true : false
+SKYDIVE_GITHUB = "/github.com/skydive-project/skydive"
+SYNCED_FOLDER = ENV.has_key?("GOPATH") ? ENV.fetch("GOPATH") + "/src" + SKYDIVE_GITHUB : "../.."
+GOPATH = "/home/vagrant"
+SKYDIVE = GOPATH + "/src" + SKYDIVE_GITHUB
 
 $guest_additions_script = <<SCRIPT
-sudo dnf -y install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
-sudo dnf -y install kernel kernel-devel VirtualBox-guest-additions
```

Comment:
this need to be done in 2 steps as VirtualBox-guest-additions is from rpmfusion

Reply:
ok, will revert